### PR TITLE
sys-apps/netloc: incompatible with >=hwloc-2.0

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
see https://www.open-mpi.org/software/netloc/v0.5/

Package-Manager: Portage-2.3.78, Repoman-2.3.17
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
ping

**Comment by Alessandro-Barbieri** [comment]
@ultrabug ping

File: sys-apps/netloc/netloc-0.5.ebuild [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
```
Comment (ultrabug) [inline sys-apps/netloc/netloc-0.5.ebuild:1]: 2020 please (repoman full should tell you usually)

**Comment by ultrabug** [comment]
@Alessandro-Barbieri sorry seems like copyright passed already, can you rebase?

**Comment by ultrabug** [comment]
Cheers

**Pull request closed without merging**